demo/src/components/heatmap/DataGrid.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
A top-level comment overviewing this component would help me.

Reply:
Me too =).  I just copied this from somewhere else.  I'll try to give some info from what I could figure out.

Reply:
Ah, please link to wherever you copied it from at the least.

Reply:
I added some comments, and there's a link to where I copied it from in `ORIGINAL_LICENSE`.